dev-python/pythondialog/pythondialog-3.3.0.ebuild

```diff
@@ -1,8 +1,8 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-PYTHON_COMPAT=( python3_6 pypy3 )
+EAPI=7
```

Comment:
EAPI change requires a revbump. Just modify PYTHON_COMPAT and nothing else.

Reply:
I think I tried that at the beginning, but `repoman` was complaining; so, I ended up having to bump the EAPI for those older versions.